src/rules/newline-after-import.js

```diff
@@ -193,11 +195,13 @@ module.exports = {
         }
       },
       'Program:exit'() {
-        log('exit processing for', context.getPhysicalFilename ? context.getPhysicalFilename() : context.getFilename());
-        const scopeBody = getScopeBody(context.getScope());
-        log('got scope:', scopeBody);
+        log('exit processing for', getPhysicalFilename(context));
 
         requireCalls.forEach((node, index) => {
+          // todo: this probably isn't correct...
+          const scopeBody = getScopeBody(getScope(context, node));
+          log('got scope:', scopeBody);
```

Comment:
the scope is for the program, i think, so it should be gotten without the node, or with the entire program's node?

Reply:
Sweet, I'll try passing in the program node and see what happens

Reply:
i think this still needs addressing